Source/Core/DolphinQt/EmulatedUSB/WiiSpeakWindow.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2025 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <QWidget>
+
+#include "Common/CommonTypes.h"
+#include "Core/Core.h"
+
+class QCheckBox;
+class QComboBox;
+class QGroupBox;
+class QLabel;
+
+class WiiSpeakWindow : public QWidget
+{
+  Q_OBJECT
+public:
+  explicit WiiSpeakWindow(QWidget* parent = nullptr);
+  ~WiiSpeakWindow() override;
+
+private:
+  void CreateMainWindow();
+  void OnEmulationStateChanged(Core::State state);
+  void EmulateWiiSpeak(bool emulate);
+  void SetWiiSpeakConnectionState(bool connected);
+  void OnInputDeviceChange();
+
+  QCheckBox* m_checkbox_enabled;
+  QComboBox* m_combobox_microphones;
+  QGroupBox* m_config_group;
+  QLabel* m_slider_label;
```

Comment:
`m_config_group` and `m_slider_label` don't need to be member variables. They are just used in one function.